lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -572,17 +572,25 @@ bool showAllSpotsBelowLine(FlSpot spot) {
   return true;
 }
 
-/// Holds data for showing dots on the bar line.
-class FlDotData {
+typedef GetDotColorCallback = Color Function(FlSpot);
```

Comment:
Please write a documentation for this too.

Reply:
fixed